index.html

```diff
@@ -1,59 +1,425 @@
-    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
-    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Harshika's Portfolio</title>
+  <link rel="stylesheet" href="style.css" class="css">
+  <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
 </head>
 <body>
-    <header>
+  <!---Navigation-->
+  <section class="imagecontainer">
+ <header>
+  <nav>
+    <div class="mobile-logo">
+      <a href="#">Welcome</a>
+    </div>
+
+    <!--hamburger-->
+    <div class="mobilemenu">
+            
+      <a href="#hamburger-icon" onclick="menuFunction()" class="hamburger"><i class="fa-sharp fa-solid fa-bars"></i> </a>
+       </div>
+       
+    <div class="inside-auto">
+  <div class="navbar-container">
+   <a href="#Portfolio">Portfolio</a>
+   <a href="#About">About</a>
+   <a href="#Contact">Contact</a>
+  </div>
+</div>
+
+  </nav>
+ </header>
+  </section>
+
+<!--Intro-->
+<section>
+<div class="infocontainer">
+  <div class="inside-auto">
+    <div class="head"><h1>Hey there.I am Harshika.</h1></div>
+    <div class="profession"><h1>I'm a software developer.</h1></div></div>
+  <div class="auto-para">
+    <div class="paragraph">
+      <p class="para"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+    </div>
+  </div>
+
+  <div class="social-media-icons">
+    <ul class="social-icons">
+      <li class="icon"><a class="fa-brands fa-twitter" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-linkedin" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-medium" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-github" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-angellist" href="#"></a> </li>
+  </ul>
+  </div>
+  </div>
+</section>
+<!---down arrow-->
+<div class="downarrow" style="margin-left:444px">
+  <img src="image/Disabled.png" alt="downarrow">
+</div>
+
+<!---Recent work-->
+<section>
+  <div class="recentworkcontainer">
+    <div class="insideautorecent">
+      <div class="headwork">
+      <h1 class="titlerecent">My Recent Work</h1>
+    </div>
+  </div>
+  </div>
+
+</section>
 
-    <nav class="mobileinfo">
-        <a class="mobilelogo"><h1>Welcome</h1></a>
+<!--recentimage container-->
+<section class="recentimagecontainer">
+  <div class="insideautoimage">
+
+  <div class="imgpatholder">
+<div class="imagebox">
+  <img src="image/Img Plaholder.png" alt="placeholder" >
+
```

Comment:
- [ ] Please match the style of your cards to the given design.

## Figma Design
![image](https://user-images.githubusercontent.com/92215878/220711823-25818832-1ca8-4da1-a06d-ce1071985294.png)


## Your Design
![image](https://user-images.githubusercontent.com/92215878/220711674-e4c84475-111d-4934-8476-605b52cab3f3.png)